src/apis/memberships.js

```diff
@@ -10,14 +9,14 @@
  *
  * @module memberships
  */
-export default (state) => {
+export default state => {
   const memberships = {}
 
-  const ROLES = memberships.ROLES = {
+  const ROLES = (memberships.ROLES = {
     READ: 'read',
     WRITE: 'write',
     ADMIN: 'admin'
-  }
+  })
```

Comment:
could you change that bit of code to following:

```javascript
const ROLES = {
  READ: 'read',
  WRITE: 'write',
  ADMIN: 'admin'
}
const memberships = { ROLES }
```

I think that's more easy on the eyes and doesn't have this weird new bracket which is added.

Reply:
fixed